database/factories/UserActivityFactory.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Database\Factories;
+use App\Models\User;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\UserActivity>
+ */
+class UserActivityFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
```

Comment:
In order to properly test this, can we add a `forUser()` method on this factory, as demonstrated in https://github.com/UnlockedLabs/UnlockEdv2/blob/5600b52402e9955b7de86b23c7880ec955da768a/database/factories/EnrollmentFactory.php#L38